test/include/_boost_pfr.cpp

```diff
@@ -0,0 +1,15 @@
+/*=============================================================================
+    Copyright (c) 2021-2023 Denis Mikhailov
```

Comment:
Do we really need these 3 new files in include directory?

Reply:
`adapted.cpp` is intended to show that there is no regression in depended on `adapted.hpp` old code. Now we have a better way for it.
`_boost_pfr.cpp` and `_boost_pfr_explicit.cpp` are intended to show that their headers don't require to include something else. I suppose there is no sense for it.

These 3 files have to be removed from this PR.